tgui/packages/tgui/interfaces/Keybinds/Keybind.tsx

```diff
@@ -19,22 +20,44 @@ interface KeybindProps {
 export const Keybind = (props: KeybindProps) => {
   const { act } = useBackend<KeybindsData>();
   const { keybind, isFocused, setFocusedKey } = props;
+
+  const [lastEvent, setLastEvent] =
+    useState<React.KeyboardEvent<HTMLDivElement>>();
```

Comment:
Probably `HTMLElement`, because button being a `div` under the hood is an implementation detail that I don't think we care about. If this makes TS angry, fine to leave as `HTMLDivElement`.